pkgs/by-name/ri/rimgo/package.nix

```diff
@@ -0,0 +1,29 @@
+{
+  lib,
+  fetchFromGitea,
+  buildGoModule,
+  tailwindcss,
+}:
+buildGoModule rec {
+  pname = "rimgo";
+  version = "1.2.0";
+  src = fetchFromGitea {
+    domain = "codeberg.org";
+    owner = pname;
+    repo = pname;
```

Comment:
```suggestion
    owner = "rimgo";
    repo = "rimgo";
```

Reply:
Alright you win.
Some packages do seem to reuse the pname attribute if possible, but out of 12 relatively randomly picked packages that could have reused the pname attribute, only [one](https://github.com/NixOS/nixpkgs/blob/nixos-23.05/pkgs/development/tools/misc/whatstyle/default.nix) did.
Statistically, the standard deviation for my experiment would be way too high to come to any conclusion, but whatever.